rmf_traffic_editor/gui/model.cpp

```diff
@@ -87,6 +87,18 @@ void Model::from_yaml(
     is_static = data["static"].as<bool>();
   else
     is_static = true;
+
+  if (data["dispensable"])
+  {
+    is_dispensable = data["dispensable"].as<bool>();
+  }
+  else
+  {
+    qWarning(
+      "parsed a deprecated .building.yaml, models should have dispensable "
+      "defined.");
```

Comment:
Wondering if this can be silenced or at least only printed once? I saw a common way to only print once in Gazebo that is to include a `static bool` variable in the `else` statement that is set when the warning is printed, although it would have the drawback of persisting through multiple map reloads without application restart.
It seems the new field is mostly needed moving forward so behavior shouldn't be broken if the map is missing the field? Loading a map with a lot of models (i.e. airport_terminal in `rmf_demos`) now prints a lot of warnings that pollute output quite a bit.

Reply:
:thinking: At the same time, the default value does not really break any behavior, you are right. There should be no harm in not printing this warning, as long as any future save will save that parameter to be used in the future.

Reply:
Removed in https://github.com/open-rmf/rmf_traffic_editor/pull/436/commits/88c9d3c61389b063e36aea017c7d746c46f362c6